physics/rrtmgp_lw_cloud_optics.F90

```diff
@@ -391,171 +376,4 @@ subroutine rrtmgp_lw_cloud_optics_init(nrghice, mpicomm, mpirank, mpiroot,
     call check_error_msg('lw_cloud_optics_init',lw_cloud_props%set_ice_roughness(nrghice))
  
   end subroutine rrtmgp_lw_cloud_optics_init
-  ! ######################################################################################
-!> \section arg_table_rrtmgp_lw_cloud_optics_run
-!! \htmlinclude rrtmgp_lw_cloud_optics.html
-!!
-!> \ingroup rrtmgp_lw_cloud_optics
-!!
-!! Compute longwave optical prperties (optical-depth) for ALL cloud types visible to RRTMGP.                                                                                                                                   
-!!
-!! \section rrtmgp_lw_gas_optics_run
-  subroutine rrtmgp_lw_cloud_optics_run(doLWrad, doG_cldoptics, icliq_lw, icice_lw,      &
```

Comment:
I can't find where this code was moved to. Was it removed?

Reply:
rrtmgp_lw_main?

Reply:
Yes. The rrtmgp_lw_cloud_optics_run routine is now in [rtmgp_lw_main.F90](https://github.com/dustinswales/ccpp-physics/blob/rrtmgp_refactor/physics/rrtmgp_lw_main.F90#L362)
Similarly, the rrtmgp_lw_gas_optics_run routine is called just before the cloud-optics in the link above.